Src/BootCamp.Chapter/BalanceStats.cs

```diff
@@ -7,31 +7,217 @@ public static class BalanceStats
         /// </summary>
         public static string FindHighestBalanceEver(string[] peopleAndBalances)
         {
-            return "";
+            if (peopleAndBalances == null || peopleAndBalances.Length == 0)
+            {
+                return "N/A.";
+            }
+
+            
+
+            var firstAccount = peopleAndBalances[0].Split(", ");
+            var firstBalance = Lesson7.StringArrayToFloatArray(firstAccount[1..]);
+
+            var names = new string[] {firstAccount[0]};
+            var highestBalance = Lesson7.MaxFloat(firstBalance);
```

Comment:
Float is not needed. Max is enough for that function name.

Reply:
It's not needed, because you can deduct what max you get based on the input type.

Reply:
Changed MaxFloat to Max and MinFloat to Min